dbms/src/Interpreters/Join.h

```diff
@@ -192,6 +198,10 @@ class Join
         skip_wait = true;
         probe_cv.notify_all();
         build_cv.notify_all();
+        for (const auto & rf : runtime_filter_list)
+        {
+            rf->cancel(log, "Join has been cancelled.");
```

Comment:
rf don't need be cancelled if it is ready?

Reply:
The check of RF status is inside of ```cancel``` function